src/xcode/ENA/TestPlans/AllTests.xctestplan

```diff
@@ -89,83 +89,42 @@
     {
```

Comment:
why are some of the screenshot tests removed but not everyone?
And what about the tests not including "screenshot" in the name, like e.g. "ENAUITests_14_ValidateHealthCertificate"?